.github/release_pull_request_template.md

```diff
@@ -0,0 +1,42 @@
+This is the release pull request for {version} of the WooCommerce Blocks plugin.
+
+## Communication
+
+<!--
+  This section is for any notes related to communicating the release.
+  Please include any extra details with each item as needed.
+-->
+
+* [ ] What is being introduced in this release?
+* [ ]  If this release has potentially **high customer impact**, is a blog post (section) ready? Provide a link if applicable.
+* [ ] Is the **relevant developer documentation updated**? Please provide links if applicable.
+* [ ] Any special instructions or helpful notes for Happiness engineers are posted (this might not be needed for every release).
+
+## Quality
+
+<!--
+  This section is for any notes related to quality around the release.
+  Please include any extra details with each item as needed.
+-->
+
+* [ ] Have you **written tests** to cover the changes?
```

Comment:
I don't see the purpose of this section in all honesty. If we're at the point of release and tests are missing a) identifying those missing tests is going to be difficult b) who is responsible for adding missing tests?

I think at this level, _the checklist should ensure tests are passing._

Adding tests needs to be done prior, at issue/epic/feature level where appropriate and/or high risk.

Reply:
(General comment) My understanding is that some of the reasoning here is to form a process for _consolidating_ the information in a central spot for easier sighting by the core wc release lead(?) - as part of feeling good about the package being updated in core.

I'm thinking of it more like part of the final checks for us to indicate that we believe we have put sufficient tests/processes in place based on our development workflow up to this point.

Reply:
As Jason pointed out, one of the purposes of this section is to communicate the state of any automated tests for the changeset in the release and if flows _are not covered_ an opportunity to indicate what _manual_ testing has been done (or give rationale for the confidence behind releasing if needed).

> I'm thinking of it more like part of the final checks for us to indicate that we believe we have put sufficient tests/processes in place based on our development workflow up to this point.

:100: exactly.

As an example, in the [release pull request for `2.6.0`](https://github.com/woocommerce/woocommerce-gutenberg-products-block/pull/2554) I included this note with this section:

> Note: Our automated test coverage is still being improved.
> 
> - The REST API Store routes have full coverage on the routes.
> - There is very little automated test coverage of Cart and Checkout blocks.
> - There is limited coverage of other blocks.
> 
> What this means practically is that a lot of heavy lifting is being done by manual testing for now.

With that said, I think the point you raised Mike does indicate there could be some additional areas left here for leaving comments (and maybe some pointers on these points)... I'll add.